harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -253,9 +253,10 @@ def _auto_step_lr_scheduler_per_batch(
         This function aims at automatically step a LR scheduler. It should be called per batch.
         """
         if lr_scheduler._step_mode == pytorch.LRScheduler.StepMode.STEP_EVERY_BATCH:
-            lr_scheduler.step()
+            if batch_idx % lr_scheduler._frequency == 0:
```

Comment:
`batch_idx` starts from 0 and it seems to always step when it begins training. You should use `batch_idx + 1` instead.